File: 7.IoT_Protocol/HTTP/nodejs/method_post.js
const axios =  require('axios');

var random_num = Math.random(190-60)+60

axios({
    method: "post",
    url :"https://api.thingspeak.com/update" ,
    data : {
        "api_key":"VIK5XGE4D5VZFQRB",
     // "field3":"56.2",
        "field3": random_num
    }
})

.then(function(response){
    console.log(response.status);
    console.log(random_num);
})
.catch(function(error){
    console.log(error);
})

File: 7.IoT_Protocol/HTTP/python/method_post.py
import requests
import random

random_num = random.randint(60,191)

payload = {   
    "api_key": "VIK5XGE4D5VZFQRB",
    "field1" : "90",
    "field3" : random_num

}

result = requests.post("https://api.thingspeak.com/update",data=payload)


print("STATUS CODE: ",result.status_code)
print("URL",result.url)
# print("",r.text)
print(f"Random Number : ",random_num)









